app-arch/torrentzip/torrentzip-1.2.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+MY_P="trrntzip-${PV}"
+
```

Comment:
Please remove the empty line, these lone groups don't improve readability and make the ebuild hard to fit in narrow windows.